engine/src/main/java/org/terasology/engine/subsystem/lwjgl/LwjglGraphics.java

```diff
@@ -16,11 +16,7 @@
 package org.terasology.engine.subsystem.lwjgl;
 
 import org.lwjgl.LWJGLException;
-import org.lwjgl.opengl.ContextAttribs;
-import org.lwjgl.opengl.Display;
-import org.lwjgl.opengl.GL43;
-import org.lwjgl.opengl.GLContext;
-import org.lwjgl.opengl.KHRDebugCallback;
+import org.lwjgl.opengl.*;
```

Comment:
Thank you! Done. I won't submit another commit to fix this unless some other changes are needed. But now the number is at 1000 so, won't happen again.